ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/TransactionService.scala

```diff
@@ -441,12 +473,13 @@ class TransactionService(session: LedgerSession) extends LedgerTestSuite(session
       "TXNotDiscloseCreateToNonSignatory",
       "Not disclose create to non-chosen branching signatory") { context =>
       for {
-        ledger <- context.participant()
-        Vector(alice, bob) <- ledger.allocateParties(2)
+        Vector(alpha, beta) <- context.participants(2)
+        alice <- alpha.allocateParty()
+        bob <- beta.allocateParty()
         template = BranchingSignatories(false, alice, bob)
-        create <- ledger.submitAndWaitRequest(bob, template.create.command)
-        transaction <- ledger.submitAndWaitForTransaction(create)
-        transactions <- ledger.flatTransactions(alice)
+        create <- beta.submitAndWaitRequest(bob, template.create.command)
+        transaction <- beta.submitAndWaitForTransaction(create)
+        transactions <- alpha.flatTransactions(alice)
```

Comment:
Potential flakiness between `alpha` and `beta`.

Reply:
Since this tests tries do "prove non-existence" of something, it looks like it's less of a case of potential flakiness and more of a source of false positives, like in the case of https://github.com/digital-asset/daml/pull/2900#discussion_r324545682. Makes sense?

Reply:
Yes, you're right.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2900/commits/5bac758267a860375dc82870dd62815fb7a4f2be